.husky/pre-commit

```diff
@@ -1 +1,5 @@
-npx lint-staged
\ No newline at end of file
+npx lint-staged
+npx eslint .
```

Comment:
Hmm, why do we have to add `npx eslint .` here? isn't it the same with `npx lint-staged`?

Reply:
Fix it. Thanks!